src/m365/spo/commands/site/SiteProperties.ts

```diff
@@ -2,4 +2,26 @@ export interface SiteProperties {
   Status: string;
   Title: string;
   Url: string;
-}
\ No newline at end of file
+}
+
+export interface RecycleBinItemProperties{
```

Comment:
Let's add a space between the last letter and `{` for readability

Reply:
removed the RecycleBinItemProperties and I use any now